openldap/defaults.yaml

```diff
@@ -2,13 +2,16 @@
 # vim: ft=yaml
 ---
 openldap:
-  client_config: /etc/ldap.conf
+  client_config: /etc/openldap/ldap.conf
   client_pkg: openldap
   server_defaults: /etc/default/slapd
+  server_config: /etc/openldap/slapd.conf
   server_pkg: openldap
   service: slapd
-  user: openldap
-  group: openldap
+  user: ldap
+  group: ldap
+  slapd_include_dir: /etc/ldap/include
```

Comment:
When reordering this in #30, I found that there were more exact values shared between `RedHat` and `Suse` than the other two, so I used the value from those as the defaults.  So please take this line and use it for `Debian` and then use the `RedHat`/`Suse` one here.